client/merchant-feedback-prompt/style.scss

```diff
@@ -2,16 +2,17 @@
 // It will be removed once the campaign is complete (paJDYF-gvt-p2).
 
 .wcpay-merchant-feedback-prompt-wrap {
-	// Borrowed from https://github.com/woocommerce/woocommerce/blob/7e7916106abc835c16388ac24097ba390a5f8d41/plugins/woocommerce/client/admin/client/layout/transient-notices/style.scss
-	position: absolute;
-	left: $gap;
-	bottom: 100%;
-	margin-bottom: $gap-small;
-	width: auto;
-
-	// Prevent SnackbarList animations to reduce annoyance on each page transition.
-	// Required due to the React portal workaround used to render the prompt in the WC footer.
 	&.components-snackbar-list {
+		// Borrowed from https://github.com/woocommerce/woocommerce/blob/7e7916106abc835c16388ac24097ba390a5f8d41/plugins/woocommerce/client/admin/client/layout/transient-notices/style.scss
+		position: absolute;
+		left: $gap;
+		right: $gap;
```

Comment:
This addition of `right`, as well as increasing specificity to ensure `width: auto` is applied, were the main changes having an impact.